index.js

```diff
@@ -1,18 +1,26 @@
+/* eslint-disable no-sync */
 // Import Third-party Depedencies
 import cacache from "cacache";
 import lodashGet from "lodash.get";
 
+// Import Node Dependencies
+import path from "node:path";
+import fs from "node:fs";
+
 // Import Internals
 import { CACHE_PATH, CURRENT_LANG } from "./src/constants.js";
 import * as languages from "./languages/index.js";
 
 import { taggedString } from "./src/utils.js";
+import { kProjectRootDir } from "./scripts/buildDocumentation.js";
```

Comment:
Unsafe import (we should not use things not intended to be shiped in the package).